Code/Framework/AzToolsFramework/AzToolsFramework/API/PythonLoader.cpp

```diff
@@ -110,7 +152,7 @@ namespace AzToolsFramework::EmbeddedPython
                     {
                         if (eggLinkLine.compare(".") != 0)
                         {
-                            resultPaths.emplace_back(eggLinkLine);
+                            resultPathCallback(AZ::IO::Path(eggLinkLine));
```

Comment:
This can be changed to be convert the eggLinkLine to an `AZ::IO::PathView` instead of an `AZ::IO::Path`.
That would prevent any unnecessary allocations needed to copy an `AZStd::string`
```suggestion
                            resultPathCallback(AZ::IO::PathView(eggLinkLine));
```